src/main/java/com/google/devtools/build/lib/packages/TargetUtils.java

```diff
@@ -55,7 +55,9 @@ public boolean apply(String tag) {
           || tag.startsWith("supports-")
           || tag.startsWith("disable-")
           || tag.equals("local")
-          || tag.startsWith("cpu:");
+          || tag.startsWith("cpu:")
```

Comment:
I know you didn't add this, but is there any test for this case? Can't see an obvious one in the files patched by this change.

Reply:
Yeah, I think so:
https://github.com/bazelbuild/bazel/blob/1f684e1b87cd8881a0a4b33e86ba66743e32d674/src/test/java/com/google/devtools/build/lib/packages/TargetUtilsTest.java#L97